lib/utils/routers.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:pokedex/features/favorite_list/favorite_connector.dart';
+import 'package:pokedex/features/favorite_list/favorite_pokemon.dart';
+import 'package:pokedex/features/pokemon_list/pokemon_list_connector.dart';
+import 'package:pokedex/features/pokemon_list/pokemon_list_page.dart';
+import 'package:pokedex/home_page.dart';
+
+final router = GoRouter(
+  observers: [routeObservers],
+  routes: <GoRoute>[
+    GoRoute(path: '/', redirect: (_) => PokemonListConnector.route, routes: []),
+    GoRoute(
+      path: PokemonListConnector.route,
+      name: PokemonListConnector.routeName,
+      pageBuilder: (context, state) => CustomTransitionPage<void>(
+        key: state.pageKey,
+        child: const HomePage(child: PokemonListPage()),
+        transitionsBuilder: (context, anim, _, child) => FadeTransition(opacity: anim, child: child),
+      ),
+      routes: [],
```

Comment:
kindly add a todo comment that later on you will be adding sub_routes here when necessary 🙂